app/main.py

```diff
@@ -7,6 +7,3 @@ def copy_file(command: str) -> None:
     with open(source, "r") as file_in, open(destination_file, "w") as file_out:
         file_out.write(file_in.read())
     print(f"Copied contents from {source} to {destination_file}")
```

Comment:
linux cp command also is not giving any feedback, so this print is also excessive)

Reply:
done)